src/main/java/edu/harvard/iq/dataverse/authorization/users/AuthenticatedUser.java

```diff
@@ -147,7 +147,7 @@ void prePersist() {
     }
     
     @PostLoad
-    void postLoad() {
+    public void postLoad() {
```

Comment:
@ErykKul - thanks for this. sorry it has been sitting around for a while. I ran it by Gustavo and he suggested naming the method something like "initialize" to better reflect what it does. Other than that it's ready for QA.

Reply:
@sekmiller I have just renamed this function.